Test Results - Average Loss: 0.5151, Average Dice Score: 0.7595100%[Finetuning] Epoch: 1 Avg Loss: 0.3159[Validation] Avg Loss: 0.3143, Avg Dice Score: 0.8908100%[Finetuning] Epoch: 2 Avg Loss: 0.3069[Validation] Avg Loss: 0.3134, Avg Dice Score: 0.8909 100%||[Finetuning] Epoch: 3 Avg Loss: 0.3011[Validation] Avg Loss: 0.3122, Avg Dice Score: 0.8897 100% |[Finetuning] Epoch: 4 Avg Loss: 0.2979[Validation] Avg Loss: 0.3138, Avg Dice Score: 0.8888 100% ||[Finetuning] Epoch: 5 Avg Loss: 0.2952[Validation] Avg Loss: 0.3133, Avg Dice Score: 0.8904100%[Finetuning] Epoch: 6 Avg Loss: 0.2935[Validation] Avg Loss: 0.3157, Avg Dice Score: 0.8844 100%||[Finetuning] Epoch: 7 Avg Loss: 0.2927[Validation] Avg Loss: 0.3103, Avg Dice Score: 0.8937 100% |[Finetuning] Epoch: 8 Avg Loss: 0.2926[Validation] Avg Loss: 0.3197, Avg Dice Score: 0.8783 100%||[Finetuning] Epoch: 9 Avg Loss: 0.2895[Validation] Avg Loss: 0.3203, Avg Dice Score: 0.8799100%[Finetuning] Epoch: 10 Avg Loss: 0.2898143/143 [00:13<00:00, 10.50it/s]143/143 [00:14<00:00, 10.18it/s]143/143 [00:13<00:00, 10.43it/s]143/143 [00:13<00:00, 10.40it/s]143/143 [00:13<00:00, 10.33it/s]143/143 [00:13<00:00, 10.36it/s]143/143 [00:13<00:00, 10.33it/s]143/143 [00:13<00:00, 10.58it/s]143/143 [00:13<00:00, 10.26it/s]143/143 [00:13 00:00, 10.36it/s][Validation] Avg Loss: 0.3151, Avg Dice Score: 0.8878 /home/mikhelson/MedImSeg-Lab24/CLUE/lib/python3.8/site-packages/lightning/pytorch/core/module.py:447: You are trying to 'self.log()' but the 'self.trainer reference is not registered on the model yet. This is most likely because the model hasn't been passed to the 'Trainer'Test Results Average Loss: 0.6520, Average Dice Score: 0.7080